app/views/schools/registrations/new.html.erb

```diff
@@ -1,29 +1,39 @@
-<h2>Sign up</h2>
-
+<h2>Sign ups</h2>
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= devise_error_messages! %>
-
   <div class="field">
     <%= f.label :email %><br />
     <%= f.email_field :email, autofocus: true %>
   </div>
-
   <div class="field">
     <%= f.label :password %>
     <% if @validatable %>
-    <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
```

Comment:
Is this for inline error messages or something? The errors above should show this validation on invalid input as well.